node/package.json

```diff
@@ -19,16 +19,15 @@
     "url": "git@github.com:uber/tchannel"
   },
   "dependencies": {
-    "bufrw": "^0.9.19",
+    "bufrw": "git://github.com/raynos/bufrw.git#upgrade-error",
     "crc": "^3.2.1",
-    "error": "^6.1.0",
+    "error": "^6.2.0",
     "farmhash": "^0.2.0",
-    "is-error": "2.0.0",
-    "json-stringify-safe": "5.0.0",
-    "null-logtron": "2.2.0",
+    "json-stringify-safe": "^5.0.0",
     "readable-stream": "1.0.33",
     "ready-signal": "^1.1.1",
-    "run-parallel": "^1.1.0",
+    "rezult": "^1.0.0",
```

Comment:
so wait, that's strictly speaking a new dependency... we already had this in bufrw?